Plugins/GitUIPluginInterfaces/BuildServerIntegration/BuildInfo.cs

```diff
@@ -22,5 +22,27 @@ public enum BuildStatus
         public bool ShowInBuildReportTab { get; set; } = true;
         public string? Tooltip { get; set; }
         public string? PullRequestUrl { get; set; }
+
+        public string StatusIcon
+        {
+            get
+            {
+                switch (Status)
+                {
+                    case BuildStatus.Success:
+                        return "✔";
+                    case BuildStatus.Failure:
+                        return "❌";
+                    case BuildStatus.InProgress:
+                        return "▶️";
+                    case BuildStatus.Stopped:
+                        return "⏹️";
+                    case BuildStatus.Unstable:
+                        return "❗";
+                    default:
+                        return "❓";
```

Comment:
Where did these pictograms come from? Is this some kind of symbol font?

Reply:
These are UTF-8 characters, so the font need to handle these Unicode characters.
If not supported the "unknown char" should show up in the "traffic light" color, so not so different from now, I cannot see it as a showstopper.
If there is a way to check if a character has support for a specific Unicode char, maybe it could be replaced with a common char?
(I have no experience in saying where these specific chars are supported.)